### ISSUE TITLE
Power SMPS for OP3

### ISSUE THREAD

**Issue by waktep** [ROBOTIS-GIT/ROBOTIS-OP3#111]
can I use smps with a voltage of 19V 4.5A?

**Comment by ROBOTIS-Will** [comment]
No, you cannot use 19V as OpenCR does not regulate the input voltage to supply power to DYNAMIXEL.
https://github.com/ROBOTIS-GIT/OpenCR/issues/276
